grails-app/conf/application.yml

```diff
@@ -183,43 +190,65 @@ dataSource:
     pooled: true
     jmxExport: true
     driverClassName: com.mysql.cj.jdbc.Driver
-    username: 'openboxes'
-    password: 'openboxes'
+    dbCreate: none
     dialect: org.hibernate.dialect.MySQL57InnoDBDialect
-    logSql: false
-    formatSql: false
+    factory: org.apache.tomcat.jdbc.pool.DataSourceFactory
+    logger: com.mysql.cj.jdbc.log.StandardLogger
+    type: org.apache.tomcat.jdbc.pool.DataSource
     properties:
+        # from https://docs.grails.org/3.3.16/guide/conf.html#dataSource
         jmxEnabled: true
-        initialSize: 10
-        maxActive: 100
+        initialSize: 5
+        maxActive: 50
         minIdle: 5
         maxIdle: 25
         maxWait: 10000
-        maxAge: 14400
+        maxAge: 10 * 60000
         timeBetweenEvictionRunsMillis: 5000
         minEvictableIdleTimeMillis: 60000
         validationQuery: SELECT 1
         validationQueryTimeout: 3
         validationInterval: 15000
-        testOnBorrow: true
+        testOnBorrow: false
         testWhileIdle: true
         testOnReturn: false
-        jdbcInterceptors: ConnectionState
+        # https://tomcat.apache.org/tomcat-8.5-doc/jdbc-pool.html#JDBC_interceptors
+        jdbcInterceptors: "ConnectionState;StatementCache(max=200)"
         defaultTransactionIsolation: java.sql.Connection.TRANSACTION_READ_COMMITTED
```

Comment:
I know this wasn't a change you made, but I think we might want to set this to the default MySQL isolation level.

> The default isolation level is [REPEATABLE READ](https://dev.mysql.com/doc/refman/5.6/en/innodb-transaction-isolation-levels.html#isolevel_repeatable-read). Other permitted values are [READ COMMITTED](https://dev.mysql.com/doc/refman/5.6/en/innodb-transaction-isolation-levels.html#isolevel_read-committed), [READ UNCOMMITTED](https://dev.mysql.com/doc/refman/5.6/en/innodb-transaction-isolation-levels.html#isolevel_read-uncommitted), and [SERIALIZABLE](https://dev.mysql.com/doc/refman/5.6/en/innodb-transaction-isolation-levels.html#isolevel_serializable). For information about these isolation levels, see [Section 14.7.2.1, “Transaction Isolation Levels”](https://dev.mysql.com/doc/refman/5.6/en/innodb-transaction-isolation-levels.html).